src/game/common/audio/audioeventinfo.h

```diff
@@ -121,4 +123,72 @@ class AudioEventInfo : public MemoryPoolObject
 
 private:
     static FieldParse s_audioEventParseTable[];
+    friend class AudioEventRTS;
 };
+
+class DynamicAudioEventInfo : public AudioEventInfo
+{
+    IMPLEMENT_POOL(DynamicAudioEventInfo);
+
+    void *operator new(size_t size, void *dst) { return dst; }
+    void operator delete(void *p, void *q) {}
```

Comment:
These operators could be guarded by `#ifdef GAME_DLL`, because they are used by the placement new from Hook_Ctor only.

Reply:
Other files don't guard them.